### PR TITLE
[OSCD] Rasautou.exe LOLbin

### PR DESCRIPTION
**Pull request by Vasilisa-L**
task 69, #1014 
Attack doesn't work in Windows 10 as options '-d' and '-p' were removed in Windows 10.
Standart 4688 in WIn7 doesn't log command line, that is why only Sysymon1 is required
Do i have some way to include only Sysmon1 and not 4688?

### CODE REVIEW

**Comment by aw350m33d** [comment]
It can be noted in the `definition` attribute of the log source that it is necessary to additionally configure the extended audit of the launch of processes. 
```
The "definition" can be used to describe the log source, including some information
on the log verbosity level or configurations that have to be applied. 
It is not automatically evaluated by the converters but gives useful advice to readers 
on how to configure the source to provide the necessary events used in the detection.
```
Somewhere in the repository, I have already met such requirements. 

Additionally, you can provide a link to a special Windows 7 update that adds the ability to configure advanced auditing feature:
https://support.microsoft.com/en-au/help/3004375/microsoft-security-advisory-update-to-improve-windows-command-line-aud

**Comment by Vasilisa-L** [comment]
@aw350m33d I added definition for log source
Do i need to move the rule in windows/create_process directory?

**Comment by aw350m33d** [comment]
Yes, a good definition, but with typos! 

Let's fix typos, move it to another folder and we will already approve it. 🥳

**Comment by Vasilisa-L** [comment]
Don't you like my typos? :worried:
I fixed them, hopefully, all

**Comment by aw350m33d** [comment]
Now it's done. 💪🏻 

This rule can be merged. @yugoslavskiy

File: rules/windows/process_creation/win_rasautou_dll_execution.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
+title: DLL Execution via Rasautou.exe
+id: cd3d1298-eb3b-476c-ac67-12847de55813
+description: Detects using Rasautou.exe for loading arbitrary .DLL specified in -d option and executes the export specified in -p. 
+status: experimental
+references:
+    - https://github.com/fireeye/DueDLLigence
+    - https://www.fireeye.com/blog/threat-research/2019/10/staying-hidden-on-the-endpoint-evading-detection-with-shellcode.html
```
Comment (yugoslavskiy) [inline rules/windows/process_creation/win_rasautou_dll_execution.yml:7]: Could you plase add a link to the original LOLBAS page?

Comment (Vasilisa-L) [reply rules/windows/process_creation/win_rasautou_dll_execution.yml:8]: Done!

Comment (yugoslavskiy) [reply rules/windows/process_creation/win_rasautou_dll_execution.yml:8]: Thanks!

**Comment by yugoslavskiy** [comment]
Thank you very much!

**Review by yugoslavskiy — APPROVED** [review]
@thomaspatzke @Neo23x0 guys, I believe that this PR could be merged (:
please let us know what do you think about it if there is anything you would like to change.

**Pull request merged**